docs: update docs closes #279 (#288)
carbon-design-system/gatsby-theme-carbon: packages/example/src/pages/guides/configuration.mdx

    },
  ],
```

## Other options

- `additionalFontWeights` – add support for additional Plex font weights. Don't forget to specify italics for the additional weights if needed.

> - `mdxExtensions` – change the file extensions processed by `gatsby-mdx` defaults to both `.mdx` and `.md`

Author: ```suggestion
- `mdxExtensions` – change the file extensions processed by `gatsby-mdx` (default ['.mdx', '.md'])
```
